saos-webapp/src/test/java/pl/edu/icm/saos/webapp/judgment/search/JudgmentSearchControllerTest.java

```diff
@@ -0,0 +1,266 @@
+package pl.edu.icm.saos.webapp.judgment.search;
+
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.setup.MockMvcBuilders.webAppContextSetup;
+import static org.hamcrest.Matchers.*;
+import static org.mockito.Mockito.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.model;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.view;
+import java.util.List;
+
+import org.assertj.core.util.Lists;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.powermock.reflect.Whitebox;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Pageable;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.ContextHierarchy;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.web.context.WebApplicationContext;
+
+import pl.edu.icm.saos.common.testcommon.category.SlowTest;
+import pl.edu.icm.saos.persistence.model.CommonCourt;
+import pl.edu.icm.saos.persistence.model.SupremeCourtChamber;
+import pl.edu.icm.saos.persistence.model.SupremeCourtJudgmentForm;
+import pl.edu.icm.saos.persistence.repository.ScJudgmentFormRepository;
+import pl.edu.icm.saos.search.search.model.JudgmentSearchResult;
+import pl.edu.icm.saos.search.search.model.SearchResults;
+import pl.edu.icm.saos.webapp.WebappTestConfiguration;
+import pl.edu.icm.saos.webapp.court.CcListService;
+import pl.edu.icm.saos.webapp.court.ScListService;
+import pl.edu.icm.saos.webapp.court.SimpleDivision;
+import pl.edu.icm.saos.webapp.court.TestCourtsFactory;
+import pl.edu.icm.saos.webapp.judgment.JudgmentCriteriaForm;
+
+/**
+ * 
+ * @author Łukasz Pawełczak
+ *
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@WebAppConfiguration
+@ContextHierarchy({ 
+	@ContextConfiguration(classes = WebappTestConfiguration.class) })
+@Category(SlowTest.class)
+public class JudgmentSearchControllerTest {
+
+	
+	@Autowired
+	private WebApplicationContext webApplicationCtx;
+	
+	private MockMvc mockMvc;
+	
+	@Autowired
+	@InjectMocks
+	private JudgmentSearchController judgmentSearchController;
+	
+	@Mock
+	private ScJudgmentFormRepository scJudgmentFormRepository;
+	
+	@Mock
+	private CcListService ccListService;
+	
+	@Mock
+	private ScListService scListService;
+	
+	@Mock
+	private JudgmentWebSearchService judgmentsWebSearchService;
+	
+	@Mock
+	private SearchResults<JudgmentSearchResult> result;
```

Comment:
There is no need to have this as field in class. Just create this object in `setUp()` method without any mocking